tasks/postgres_upgrade.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+echo 'Checking pgsql version'
+CMD=$(sudo gitlab-psql --version)
```

Comment:
The `sudo` needs to come out here, and this task needs to be run with `--sudo --tty --run-as root`